src/main/kotlin/no/nav/syfo/service/SykmeldingService.kt

```diff
@@ -168,4 +189,19 @@ class SykmeldingService constructor(
                 etternavn = behandlerFraHpr.etternavn
         )
     }
+
+    fun checkIfOppgaveIsDuplikat(oppgave: OppgaveResultat, journalpostId: String?, sakid: String?, loggingMeta: LoggingMeta) {
```

Comment:
Dette blir ikke riktig, du har slått sammen to ulike metrikker til en. Fordelingsoppgave og vanlig oppgave er to ulike ting :/

Reply:
ifix